cpmpy/expressions/globalconstraints.py

```diff
@@ -455,7 +455,15 @@ def decompose(self):
         return cons
 
     def value(self):
-        start, dur, end, demand, cap = [argval(a) for a in self.args]
+        import numpy as np
+
+        argvals = [np.array([argval(a) for a in arg]) if is_any_list(arg)
+                   else argval(arg) for arg in self.args]
```

Comment:
this still does not guarantee that it is a np.array (the latter case), which you need for the .all() below to work?

if it does, please add a comment that we assume the args are a numpy array here...

Reply:
start, end and duration will always be lists (as fixed by Helge's pull request), so non-numpied args here are only the capacity and potentially the demand if fixed for all tasks.